README.md

```diff
@@ -1,5 +1,34 @@
 # Curated list of great Scala oriented (but not always) Functional Programming resources
 
+## Your personal reading list
+
+If you wish to track your progress in the reading list, follow these steps:
+
+<details>
+<summary>Steps</summary>
+
+**Create a new branch so you can check items like this, just put an x in the brackets: [x]**
+
+    Fork a branch and follow the commands below
+
+`git checkout -b my-personal-list`
+
+`git remote add mmenestret https://github.com/mmenestret/fp-resources`
+
+`git fetch --all`
+
+    Mark all boxes with X after you completed your changes
+
+`git add . `
+
+`git commit -m "Marked x" `
+
+`git rebase mmenestret/master `
+
+`git push --force `
```

Comment:
When you push changes to a remote server, Git prevents you from pushing your changes if the remote has changes you’re missing. This feature is designed to prevent data loss. Since the person making the changes on his personal branch will be alone on it. It won't do any harm to do a `git push --force`.
But, actually, it is not a good practice to use the flag `--force` and I changed the steps to `git pull` instead of `git rebase`.